### PR TITLE
PR #247 #246 #245 #244 #243

### LINKED ISSUE #247: Triple alignment-enhanced complex question answering over knowledge bases
https://www.sciencedirect.com/science/article/pii/S0925231224004806

### PR DESCRIPTION
**Pull request by LongquanJiang**
close #247 #246 #245 #244 #243

### CODE REVIEW

File: freebase/SimpleQuestions.md [hunk 1]
```diff
@@ -3,27 +3,36 @@ name: SimpleQuestions
...
+|       KGQA-RR(GPT2)        | 2023 |   -    |      -    |   -    |   -   |  23.18   |    EN    |                     [Hu et al.](https://arxiv.org/pdf/2303.10368.pdf)                      |
+|    EffiQA w/Deepseek-V2    | 2024 |  69.4  |     -     |   -    |   -   |    -     |    EN    |              [Dong et al.](https://arxiv.org/pdf/2406.01238)                 |
+|      EffiQA w/ChatGPT      | 2024 |  65.7  |     -     |   -    |   -   |    -     |    EN    |              [Dong et al.](https://arxiv.org/pdf/2406.01238)                 | 
+|       EffiQA w/GPT-4       | 2024 |  76.5  |     -     |   -    |   -   |    -     |    EN    |              [Dong et al.](https://arxiv.org/pdf/2406.01238)                 |
+|       Prior FT SOTA        | 2024 |  85.8  |     -     |   -    |   -   |    -     |    EN    |              [Dong et al.](https://arxiv.org/pdf/2406.01238)                 |
+|    Prior Prompting SOTA    | 2024 |   -    |     -     |   -    |   -   |    -     |    EN    |              [Dong et al.](https://arxiv.org/pdf/2406.01238)                 |
```
Comment (DavidBlavid) [inline freebase/SimpleQuestions.md]: This row has no evaluation value. Should we remove it?

File: systems.md [hunk 2]
```diff
@@ -141,4 +141,6 @@
...
+| Generate-onGraph (GoG)               | Xu et al.                            | [Link](https://arxiv.org/pdf/2404.14741)                                                                                                                                                                                                                                                                         | no                      | no                                                                                                                                                             | [Link](https://arxiv.org/pdf/2404.14741) |  Generate-onGraph (GoG) can effectively integrate the external and inherent knowledge of LLMs | [Link](https://arxiv.org/pdf/2404.14741) |
```
Comment (DavidBlavid) [inline systems.md]: Just a minor thing. When no Code/Demo etc. is linked, we set it to '-', not 'no'

File: wikidata/QALD-10.md [hunk 3]
```diff
@@ -3,10 +3,19 @@
...
+|      Gavrilev et al.       | 2022 |   -    | 14.21 (Macro) | 14.00 (Macro) | 19.48 (Macro) |    EN    | [GERBIL](https://gerbil-qa.aksw.org/gerbil/experiment?id=202205210032)  |
+|    EffiQA w/Deepseek-V2    | 2024 |  50.2  |       -       |       -       |       -       |    EN    |             [Dong et al.](https://arxiv.org/pdf/2406.01238)             |
+|      EffiQA w/ChatGPT      | 2024 |  46.2  |       -       |       -       |       -       |    EN    |             [Dong et al.](https://arxiv.org/pdf/2406.01238)             | 
+|       EffiQA w/GPT-4       | 2024 |  51.4  |       -       |       -       |       -       |    EN    |             [Dong et al.](https://arxiv.org/pdf/2406.01238)             |
+|       Prior FT SOTA        | 2024 |  45.4  |       -       |       -       |       -       |    EN    |             [Dong et al.](https://arxiv.org/pdf/2406.01238)             |
+|    Prior Prompting SOTA    | 2024 |   -    |       -       |       -       |       -       |    EN    |             [Dong et al.](https://arxiv.org/pdf/2406.01238)             |
```
Comment (DavidBlavid) [inline wikidata/QALD-10.md]: This row has no evaluation value. Remove?

Comment (DavidBlavid) [reply wikidata/QALD-10.md]: Also please fix ordering according to Hits@1

**Review by DavidBlavid — COMMENTED** [review]
Issues #245 #244 #243 look good. I did not find anything for Issues #247 and #246 though. Were these systems already on the leaderboard, or what is the reason they are not included?

I also marked minor formatting changes, but the content looks correct.

File: freebase/SimpleQuestions.md (hunk 1 above)
Comment (LongquanJiang) [reply freebase/SimpleQuestions.md]: Yes, already remove

File: wikidata/QALD-10.md (hunk 3 above)
Comment (LongquanJiang) [reply wikidata/QALD-10.md]: Already removed

**Pull request closed without merging**